dev.sh

```diff
@@ -129,6 +130,28 @@ function test-apps {
   _docker_compose exec task-runner npx vitest run $files
 }
 
+function dev-apps {
+  (
+    set -e  # Exit on error
```

Comment:
We could remove this comment since it doesn't add more information about the line.

```suggestion
    set -e
```